auth/src/AzureAuthentication.ts

```diff
@@ -9,12 +9,19 @@ import type * as vscode from 'vscode';
  * Represents a means of obtaining authentication data for an Azure subscription.
  */
 export interface AzureAuthentication {
+    /**
+     * Gets a VS Code authentication session for an Azure subscription.
+     * Always uses the default scope, `https://management.azure.com/.default/.default.`

```

Comment:
Clarify here that it will respect the `microsoft-sovereign-cloud.environment` setting.